### ISSUE TITLE
Cannot find module '../mantine-styles/src/tss/create-styles'

### ISSUE THREAD

**Issue by ioson** [mantinedev/mantine#619]
### What package has an issue

@mantine/core

### Describe the bug

I cannot build Mantine 3.4.3 with a default Vite React Typescript Setup. All Components throw this error on build:
`Cannot find module '../mantine-styles/src/tss/create-styles' or its corresponding type declarations.`

Tried with a fresh test install - same result.

To reproduce:
`yarn create @vitejs/app mantine-vite --template react-ts`
`yarn add dayjs @mantine/notifications @mantine/hooks @mantine/dates @mantine/core`

Edit App.tsx:
```
import { Button } from '@mantine/core';

export default function App() {
  return <Button>Hello world!</Button>;
}
```

`npm run build`

### In which browser did the problem occur

Bash

### Do you know how to fix the issue

No

**Comment by rtivital** [comment]
This issue will be fixed in 3.5, meanwhile you can set `skipLibCheck` option in your tsconfig to `false`

**Comment by ioson** [comment]
> This issue will be fixed in 3.5, meanwhile you can set `skipLibCheck` option in your tsconfig to `false`

Great, thanks. Awesome library by the way! Really enjoy using it.

**Comment by rtivital** [comment]
Fixed in 3.5.0
